examples/gsph/run_disc_1sink.py

```
@@ -0,0 +1,265 @@
+t_stop = [i * dt_stop for i in range(nstop + 1)]
+print(f"t_stop = {t_stop}")
+
+idump = 0
+istop = 0
+c = 0
+for ttarg in t_stop:
+    if ttarg >= t_start and c < 10:
+        # model.evolve_until(ttarg)
+        model.evolve_once()
+
+        model.do_vtk_dump(get_vtk_dump_name(idump), True)
```

Ruff (0.16.2)

[warning] Boolean positional value in function call

(FBT003)